### PR TITLE
Change 1.3.3 to pass for --use-service-account-credentials=true and --use-service-account-credentials

### PR DESCRIPTION
**Pull request by mwwolters**
The setting controlled by the flag --use-service-account-credentials can be enabled either through --use-service-account-credentials=true or --use-service-account-credentials, but this check fails for the second case. This change checks for that case.

#441

### CODE REVIEW

File: cfg/1.11/master.yaml [hunk 1]
```diff
@@ -544,12 +544,18 @@ groups:
...
       - flag: "--enable-admission-plugins"
         compare:
           op: has
           value: "NodeRestriction"
         set: true
+      - flag: "--admission-control"
+        compare:
+          op: has
+          value: "NodeRestriction"
+        set: true
```
Comment (lizrice) [inline cfg/1.11/master.yaml]: I think this change is something else

File: cfg/1.13/master.yaml [hunk 2]
```diff
@@ -548,12 +548,18 @@ groups:
...
       - flag: "--enable-admission-plugins"
         compare:
           op: has
           value: "NodeRestriction"
         set: true
+      - flag: "--admission-control"
+        compare:
+          op: has
+          value: "NodeRestriction"
+        set: true
```
Comment (lizrice) [inline cfg/1.13/master.yaml]: Also something else, I think

File: cfg/1.11/master.yaml (hunk 1 above)
Comment (mwwolters) [reply cfg/1.11/master.yaml]: Whoops, you're right. I think it should be fixed now.

File: cfg/1.13/master.yaml (hunk 2 above)
Comment (mwwolters) [reply cfg/1.13/master.yaml]: Same as above.

**Pull request merged**